app/controllers/application_controller.rb

```diff
@@ -2,4 +2,11 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  helper_method :current_user
+
+  private
+
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
```

Comment:
Just so you know if you want to precent an `ActiveRecord::RecordNotFound` from being raised you can also do `User.find_by(id: session[:user_id])`.